src/components/Pagination/Pagination.stories.mdx

```diff
@@ -0,0 +1,44 @@
+import {
+  Meta,
+  Story,
+  ArgsTable,
+  Preview,
+  Description,
+} from "@storybook/addon-docs/blocks";
+import { withDesign } from "storybook-addon-designs";
+import * as stories from "./Pagination.stories.tsx";
+
+<Meta
+  title="Pagination"
+  component={Pagination}
+  decorators={[withDesign]}
+  parameters={{
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=17226%3A932",
+    },
+  }}
+/>
+
+# Pagination
+
+<Description of={Pagination} />
+Pagination navigates between the pages of a multi-page application. It is commonly
```

Comment:
The paragraph under "description" is not showing up in the storybook preview - anyone know why this is happening?

Reply:
Per discussion, this issue will be resolved under a separate documentation-specific ticket.